test/test_util.py

```diff
@@ -130,3 +130,16 @@ def test_is_pkcs12(self):
             'rsa_2048_key_cert_encrypted.pfx') as \
                 rsa_2048_key_cert_encrypted_pfx:
             self.assertTrue(is_pkcs12(rsa_2048_key_cert_encrypted_pfx.read()))
+
+    def test_form_factor_from_code(self):
+        self.assertEqual(FORM_FACTOR.UNKNOWN, FORM_FACTOR.from_code(None))
+        self.assertEqual(
+            FORM_FACTOR.UNKNOWN, FORM_FACTOR.from_code('im a string'))
```

Comment:
I feel like calling `from_code` with a string argument should probably raise an exception instead - because that should never happen, right?

Reply:
Sure, that's true. I'll update the PR.